.github/workflows/release.yml

```diff
@@ -60,23 +57,21 @@ jobs:
           # Make the commit
           MESSAGE="Build for $(git rev-parse --short HEAD)"
           git commit --allow-empty -m "$MESSAGE"
+          git tag -f -a -m "Release $longVersion" $longVersion
 
-          # Create an annotated tag and push it to origin. Using -f to overwrite
-          # the tag that `npm version` made for us in a previous step
-          git tag -f -a -m "Release $TAG" $TAG
-          git push origin $TAG
-
-  release:
-    name: Release
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Create Release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ needs.build.outputs.tag }}
-          release_name: Release ${{ needs.build.outputs.tag }}
-          draft: true
-          prerelease: false
+          # Get the commit of the tag you just released
+          commitHash=$(git rev-list -n 1 $longVersion)
+          
+          # Delete the old major and minor version tags locally
+          git tag -d $majorVersion || true
+          git tag -d $majorVersion || true
+          
+          # Make new major and minor version tags locally that point to the commit you got from the "git rev-list" above
+          git tag -f $majorVersion $commitHash
+          git tag -f $minorVersion $commitHash
+          
+          # Force push the new minor version tag to overwrite the old tag remotely
+          echo "Pushing new tags"
+          git push -f origin $longVersion
```

Comment:
Why do we have to overwrite the original tag? Is this because the package.json version is changing? How do we keep the package.json version consistent with our release version on the main branch?

Reply:
We need to push the built `dist` and `lib` folders to the new release tag. I did add a change to bump package.json in main as well.